cluster/manifests/kube-cluster-autoscaler/daemonset.yaml

```diff
@@ -31,7 +33,7 @@ spec:
         effect: NoSchedule
       containers:
       - name: cluster-autoscaler
-        image: registry.opensource.zalan.do/teapot/kube-cluster-autoscaler:{{if eq .Cluster.ConfigItems.experimental_autoscaler_1_14 "true" }}{{ .Cluster.ConfigItems.experimental_autoscaler_1_14_version }}{{else}}v1.12.2-internal.4{{end}}
+        image: registry.opensource.zalan.do/teapot/kube-cluster-autoscaler:{{ .Cluster.ConfigItems.cluster_autoscaler_version }}
```

Comment:
Do we still want to keep it customisable if we don't have the settings for the 1.14 version?

Reply:
removed the config item